cfme/common/provider.py

```diff
@@ -167,6 +167,14 @@ def get_yaml_data(self):
             raise ProviderHasNoKey(
                 'Provider {} has no key, so cannot get yaml data'.format(self.name))
 
+    def has_nodes(self):
```

Comment:
Well if you have normal provides, it is hosts, if you have OS infra it is nodes ...